diff -urN linux-2.6.23-imedia.orig/drivers/media/video/ivtv/ivtv-i2c.c linux-2.6.23-imedia/drivers/media/video/ivtv/ivtv-i2c.c
--- linux-2.6.23-imedia.orig/drivers/media/video/ivtv/ivtv-i2c.c	2007-11-20 11:58:06.000000000 +0200
+++ linux-2.6.23-imedia/drivers/media/video/ivtv/ivtv-i2c.c	2007-11-20 11:58:16.000000000 +0200
@@ -109,7 +109,6 @@
 	I2C_DRIVERID_UPD64083,
 	I2C_DRIVERID_SAA717X,
 	I2C_DRIVERID_WM8739,
-	I2C_DRIVERID_VP27SMPX,
 	0 		/* IVTV_HW_GPIO dummy driver ID */
 };
 
diff -urN linux-2.6.23-imedia.orig/drivers/media/video/vp27smpx.c linux-2.6.23-imedia/drivers/media/video/vp27smpx.c
--- linux-2.6.23-imedia.orig/drivers/media/video/vp27smpx.c	2007-11-20 11:59:19.000000000 +0200
+++ linux-2.6.23-imedia/drivers/media/video/vp27smpx.c	2007-11-20 11:59:05.000000000 +0200
@@ -191,7 +191,6 @@
 	.driver = {
 		.name = "vp27smpx",
 	},
-	.id             = I2C_DRIVERID_VP27SMPX,
 	.attach_adapter = vp27smpx_probe,
 	.detach_client  = vp27smpx_detach,
 	.command        = vp27smpx_command,
