From auke-jan.h.kok@intel.com Mon Aug 27 16:16:23 2007
From: Auke Kok <auke-jan.h.kok@intel.com>
Date: Mon, 27 Aug 2007 16:16:13 -0700
Subject: USB: AMD5536: use pdev->revision
To: gregkh@suse.de
Cc: thomas.dahlmann@amd.com, linux-pci@atrey.karlin.mff.cuni.cz
Message-ID: <20070827231605.27448.62875.stgit@localhost.localdomain>


Signed-off-by: Auke Kok <auke-jan.h.kok@intel.com>
Cc: Thomas Dahlmann <thomas.dahlmann@amd.com>
Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>

---
 drivers/usb/gadget/amd5536udc.c |    6 ++----
 1 file changed, 2 insertions(+), 4 deletions(-)

--- a/drivers/usb/gadget/amd5536udc.c
+++ b/drivers/usb/gadget/amd5536udc.c
@@ -3285,14 +3285,12 @@ static int udc_pci_probe(
 
 	pci_set_drvdata(pdev, dev);
 
-	/* chip revision */
-	dev->chiprev = 0;
+	/* chip revision for Hs AMD5536 */
+	dev->chiprev = pdev->revision;
 
 	pci_set_master(pdev);
 	pci_set_mwi(pdev);
 
-	/* chip rev for Hs AMD5536 */
-	pci_read_config_byte(pdev, PCI_REVISION_ID, (u8 *) &dev->chiprev);
 	/* init dma pools */
 	if (use_dma) {
 		retval = init_dma_pools(dev);
