From 7098428487da762d226bbc070055a3ecbb7a5990 Mon Sep 17 00:00:00 2001
From: Dave Airlie <airlied@redhat.com>
Date: Tue, 28 Jul 2009 03:32:37 +0000
Subject: [PATCH 09/13] sis: update for resources/RAC API removal

---
 src/sis.h        |    2 ++
 src/sis_driver.c |    7 +++++++
 2 files changed, 9 insertions(+), 0 deletions(-)

diff --git a/src/sis.h b/src/sis.h
index 879ca9b..f1f6eb0 100644
--- a/src/sis.h
+++ b/src/sis.h
@@ -64,7 +64,9 @@
 #include "xf86Pci.h"
 #include "xf86Priv.h"
 #include "xf86_OSproc.h"
+#ifndef XSERVER_LIBPCIACCESS
 #include "xf86Resources.h"
+#endif
 #include "xf86.h"
 #include "xf86PciInfo.h"
 #include "xf86Cursor.h"
diff --git a/src/sis_driver.c b/src/sis_driver.c
index 373eefb..9fe2621 100644
--- a/src/sis_driver.c
+++ b/src/sis_driver.c
@@ -78,7 +78,9 @@
 
 #include "sis.h"
 
+#ifndef XSERVER_LIBPCIACCESS
 #include "xf86RAC.h"
+#endif
 #include "dixstruct.h"
 #include "shadowfb.h"
 #include "fb.h"
@@ -3518,6 +3520,7 @@ SISPreInit(ScrnInfoPtr pScrn, int flags)
     SiS_MapVGAMem(pScrn);
 #endif
 
+#ifndef XSERVER_LIBPCIACCESS
     /* Set operating state */
 
     /* 1. memory */
@@ -3547,6 +3550,8 @@ SISPreInit(ScrnInfoPtr pScrn, int flags)
     /* Operations for which I/O access is required */
     pScrn->racIoFlags = RAC_COLORMAP | RAC_CURSOR | RAC_VIEWPORT;
 
+#endif
+
     /* Load ramdac module */
     if(!xf86LoadSubModule(pScrn, "ramdac")) {
        SISErrorLog(pScrn, "Could not load ramdac module\n");
@@ -4321,6 +4326,7 @@ SISPreInit(ScrnInfoPtr pScrn, int flags)
     xf86DrvMsg(pScrn->scrnIndex, from, "MMIO registers at 0x%lX (size %ldK)\n",
 	   (ULong)pSiS->IOAddress, pSiS->mmioSize);
 
+#ifndef XSERVER_LIBPCIACCESS
     /* Register the PCI-assigned resources */
     if(xf86RegisterResources(pSiS->pEnt->index, NULL, ResExclusive)) {
        SISErrorLog(pScrn, "PCI resource conflicts detected\n");
@@ -4332,6 +4338,7 @@ SISPreInit(ScrnInfoPtr pScrn, int flags)
        SISFreeRec(pScrn);
        return FALSE;
     }
+#endif
 
     from = X_PROBED;
     if(pSiS->pEnt->device->videoRam != 0) {
-- 
1.7.1

