From b942a9351b3dfe0e44864965f1fd3d7f47e14e05 Mon Sep 17 00:00:00 2001
From: Adam Jackson <ajax@redhat.com>
Date: Tue, 18 May 2010 17:11:01 +0000
Subject: [PATCH 12/13] Remove mibank.h reference

Signed-off-by: Adam Jackson <ajax@redhat.com>
---
 src/sis_driver.c |    1 -
 1 files changed, 0 insertions(+), 1 deletions(-)

diff --git a/src/sis_driver.c b/src/sis_driver.c
index 81b6276..3734684 100644
--- a/src/sis_driver.c
+++ b/src/sis_driver.c
@@ -85,7 +85,6 @@
 #include "shadowfb.h"
 #include "fb.h"
 #include "micmap.h"
-#include "mibank.h"
 #include "mipointer.h"
 #include "mibstore.h"
 #include "edid.h"
-- 
1.7.1

