From 13583aba3e2abafd55fc894070cf5f050bbea4f2 Mon Sep 17 00:00:00 2001
From: Adam Jackson <ajax@redhat.com>
Date: Tue, 18 May 2010 17:11:01 +0000
Subject: Remove mibank.h reference

Signed-off-by: Adam Jackson <ajax@redhat.com>
---
diff --git a/src/sis_driver.c b/src/sis_driver.c
index d19cf17..22646d2 100644
--- a/src/sis_driver.c
+++ b/src/sis_driver.c
@@ -56,7 +56,6 @@
 #include "shadowfb.h"
 #include "fb.h"
 #include "micmap.h"
-#include "mibank.h"
 #include "mipointer.h"
 #include "mibstore.h"
 #include "edid.h"
--
cgit v0.8.3-6-g21f6
