From 387b3b52f1c03b3eb16d60c3674d7e6c59e03028 Mon Sep 17 00:00:00 2001
From: Paulo Zanoni <pzanoni@mandriva.com>
Date: Wed, 9 Feb 2011 09:31:34 -0200
Subject: [PATCH 13/13] Update to new CreateNewResourceType API

---
 src/sis_mergedfb.c |    4 ++--
 1 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/src/sis_mergedfb.c b/src/sis_mergedfb.c
index feca024..34a0ea3 100644
--- a/src/sis_mergedfb.c
+++ b/src/sis_mergedfb.c
@@ -2948,11 +2948,11 @@ SiSXineramaExtensionInit(ScrnInfoPtr pScrn)
 
        while(SiSXineramaGeneration != serverGeneration) {
 
-	  ClientType = CreateNewResourceType(SiSXineramaFreeClient);
+	  ClientType = CreateNewResourceType(SiSXineramaFreeClient, "sisimediaxinerama");
 	  if(!ClientType)
 	     break;
 
-	  EventType = CreateNewResourceType(SiSXineramaFreeEvents);
+	  EventType = CreateNewResourceType(SiSXineramaFreeEvents, "sisimediaxinerama");
 	  if(!EventType)
 	     break;
 
-- 
1.7.1

